packages/docs/src/components/layout/SideNav/ThemeVersionSection.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useRef, useState } from 'react';
+import useTheme from '../../../helpers/useTheme';
+import ThemeDialog from './ThemeDialog';
+import VersionDialog from './VersionDialog';
+import { getPackageData, getThemeData } from './themeVersionData';
+import { ArrowIcon } from '@cmsgov/design-system';
+import { useFilterDialogManager } from '../FilterDialog/FilterDialogManager';
+
+export const ThemeVersionSection = () => {
+  const theme = useTheme();
+  const version = getPackageData(theme).version;
+  const themeDisplayName = getThemeData(theme).displayName;
+  const themeDialogProps = useFilterDialogManager();
+  const versionDialogProps = useFilterDialogManager();
+  const themeButtonRef = useRef<HTMLButtonElement>();
+  const versionButtonRef = useRef<HTMLButtonElement>();
+
+  return (
+    <>
+      <div>
+        Theme:{' '}
+        <button
+          className="c-navigation__settings-button"
+          onClick={themeDialogProps.toggleClick}
+          type="button"
+          ref={themeButtonRef}
+        >
+          {themeDisplayName} <span className="ds-u-visibility--screen-reader">(Change theme)</span>{' '}
+          <ArrowIcon direction="down" />
+        </button>
+        {themeDialogProps.isOpen && (
+          <ThemeDialog
+            theme={theme}
+            onExit={() => {
+              themeDialogProps.closeClick();
+              themeButtonRef.current?.focus();
```

Comment:
@zarahzachz, for some reason, I have to manually focus back on this element after closing the dialog, and I haven't gotten to the bottom of it. This could also be affecting the drawer component, but we just need the right scenario to reproduce it. I can't seem to find documentation on the html `<dialog>` element's behavior on restoring focus after closing.